titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/validator/PassJobValidator.java

```diff
@@ -10,7 +26,7 @@
 import reactor.core.publisher.Mono;
 
 /**
- * This {@link EntityValidator} implementation always causes validation to fail.  It is used as a default implementation which
+ * This {@link EntityValidator} implementation always causes validation to pass.  It is used as a default implementation which
  * should be overriden.
```

Comment:
overriden -> overridden

Reply:
Not my typo :-) but changed anyways. I suppose I did miss IntelliJ's squiggly spelling line.